mlrun/api/db/sqldb/db.py

```diff
@@ -1181,6 +1181,36 @@ def list_entities(
                 )
         return schemas.EntitiesOutput(entities=entities_results)
 
+    @staticmethod
+    def _assert_group_by_parameters(group_by, sort):
+        if sort is None:
+            raise mlrun.errors.MLRunInvalidArgumentError(
+                "sort parameter must be provided when group_by is used."
+            )
```

Comment:
I think we can have a sane default to `updated` (when group_by is given)

Reply:
We agreed to not provide a default for the sort field (we may want to add `created` shortly). I would prefer keeping it without a default and having the user specify the value.